Add tempo start docs (#4984)
grafana/tempo: docs/sources/tempo/setup/command-line-flags.md

| `--target` | Target module to run | `all` |

## Authentication and multitenancy

| Flag | Description | Default |
| --- | --- | --- |
| `--auth.enabled` | Set to true to enable auth (deprecated: use multitenancy.enabled) | `false` |
| `--multitenancy.enabled` | Set to true to enable multitenancy | `false` |

## HTTP and API settings

| Flag | Description | Default |
| --- | --- | --- |
| `--http-api-prefix` | String prefix for all HTTP API endpoints | `""` |
| `--enable-go-runtime-metrics` | Set to true to enable all Go runtime metrics | `false` |
| `--shutdown-delay` | How long to wait between SIGTERM and shutdown | `0` |

## Server settings

| Flag | Description | Default |
| --- | --- | --- |
| `--server.http-listen-port` | HTTP server listen port | `80` |
| `--server.grpc-listen-port` | gRPC server listen port | `9095` |

## Memberlist settings

| Flag | Description | Default |
| --- | --- | --- |
| `--memberlist.host-port` | Host port to connect to memberlist cluster | |
| `--memberlist.bind-port` | Port for memberlist to communicate on | `7946` |
| `--memberlist.message-history-buffer-bytes` | Size in bytes for the message history buffer | `0` |

## Module configuration

> Additional flags exist for configuring individual Tempo modules (distributor, ingester, querier, etc.) and their components. These flags follow a pattern like `--<module>.<setting>` and are extensively documented in the configuration file format.

Reviewer: Update to remove passive voice.
```suggestion
You can use additional flags to configuring individual Tempo modules, such as the distributor, ingester, querier, and their components. These flags follow a pattern like `--<module>.<setting>` and are extensively documented in the configuration file format.
```
